Lib/fontTools/varLib/merger.py

```diff
@@ -900,7 +897,6 @@ def merge(merger, self, lst):
 	self.__dict__ = lst[0].__dict__.copy()
 
 	instancer = merger.instancer
-	# TODO Handle differing valueformats
```

Comment:
this comment was a copy-paste leftover. There are no "differing valueformats" across multiple fonts since the MutatorMerger is called with a single font (see merger.mergeTables() second parameter).